editor/src/components/navigator/navigator.spec.browser2.tsx

```diff
@@ -9,14 +9,10 @@ import { offsetPoint, windowPoint, WindowPoint } from '../../core/shared/math-ut
 import { PrettierConfig } from 'utopia-vscode-common'
 import * as Prettier from 'prettier/standalone'
 import { BakedInStoryboardVariableName, BakedInStoryboardUID } from '../../core/model/scene-utils'
-import { CSSCursor } from '../canvas/canvas-types'
 import { getDomRectCenter } from '../../core/shared/dom-utils'
 import { selectComponents } from '../editor/actions/action-creators'
 import * as EP from '../../core/shared/element-path'
-
-interface CheckCursor {
-  cursor: CSSCursor | null
-}
+import { wait } from '../../core/model/performance-scripts'
```

Comment:
unused import

Reply:
removed in https://github.com/concrete-utopia/utopia/pull/3256/commits/3b61a73535dec9b44a4223aaf271fa3775d9deba